Find a functionally equivalent code:com.android.internal.view.InputBindResult.writeToParcel:COMMENT
Method Modifier: public      internal    
Comment:/**
 * Used to package this object into a {@link Parcel}.
 *
 * @param dest The {@link Parcel} to be written.
 * @param flags The flags used for parceling.
 */

@@ -8,11 +8,12 @@
     }
     dest.writeString(id);
     dest.writeInt(sequence);
+    dest.writeInt(userActionNotificationSequenceNumber);
 }

